packages/plugins/babel/__tests__/__fixtures__/resolver/index.js

```diff
@@ -0,0 +1,11 @@
+const { join } = require('path');
+
+module.exports = {
+  resolveSync(_, filename, importPath) {
```

Comment:
This shouldn't have the same API as `enhanced-resolve` because it will always be `{}`. It should just be `filename` and `importPath` so you'll need to modify `createCustomResolver` to have the same contract.

Reply:
true, but I do not want here to change how existing API works or touch `createCustomResolver`. I'm just ignoring the first argument since it's not need by custom Atlassian resolver, but it might be needed by some other resolver implementation

fewer changes in existing code, fewer tests to fix, etc, etc

Reply:
Don't love that as a reason, but we could make use of it later (but in reality probably not)